pkg/apis/pipeline/v1alpha1/verificationpolicy_types.go

```diff
@@ -62,6 +62,10 @@ type VerificationPolicySpec struct {
 	Resources []ResourcePattern `json:"resources"`
 	// Authorities defines the rules for validating signatures.
 	Authorities []Authority `json:"authorities"`
+	// Mode controls whether a failing policy will fail the taskrun/pipelinerun, or only log the warnings
```

Comment:
are there docs we need to update with the new Mode field?

Reply:
Not yet, I plan to use this new field in next pr(and add docs there) to keep the PR small.  I think we should merge these 2 PRs in one release. So better add this PR to v0.47

Reply:
Tracking issue here: https://github.com/tektoncd/pipeline/issues/6356